test-e2e/fixtures/config-test-fixtures.ts

```diff
@@ -108,4 +119,21 @@ export const testSubdomainRouting = base.extend<{ rootDomain: string, baseURL: s
   }
 })
 
+/**
+ * A fixture that skips tests that require the service worker. This is needed in order to test handling of requests where the service worker is not present.
+ *
+ * @see https://github.com/ipfs/service-worker-gateway/issues/272
+ */
+export const testNoServiceWorker = base.extend<{ rootDomain: string, baseURL: string, protocol: string }>({
```

Comment:
not extending our `test` fixture above because that is trying to skip everything that skips tests on the `isNoServiceWorkerProject(test)` check. we need the opposite here.